Zinc/Private/ZincRepo.m

```diff
@@ -34,8 +34,10 @@
 #define DOWNLOADS_DIR @"zinc/downloads"
 #define REPO_INDEX_FILE @"repo.json"
 
-static NSMutableDictionary* _ReposByURL;
+static const float kContentBundleFlushLimitInMegabytes = 100.f;
+static NSString * const kContentBundlePrefix = @"com.wonder.content";
```

Comment:
same for `kContentBundleFlushLimitInMegabytes` above , the regex `@"\\A(com\\.wonder\\.content\\d*\\..*)-(.*)\\Z";`, and everything else in this PR that's named `content`...will take a bit of work just to make this all very generic.